### DISCUSSION TITLE
Icon request: fa-wardrobe

### DISCUSSION THREAD

**Discussion by albertobissacco** [FortAwesome/Font-Awesome#21376] [Icon Requests]
Would be very useful have a wardrobe icon like this
![wardrobe](https://cloud.githubusercontent.com/assets/189348/3811008/070ba30e-1c99-11e4-8b0d-80ff82020cf1.png)

**Comment by brainscript** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by sensibleworld** [comment]
Hey there!

We've made the decision to move our icon requests flow to **GitHub Discussions**, as opposed to issue tracking. No more +1s or thumbs-ups needed. There's an upvote system built in.

As part of this transition, we are closing existing icon requests with lower comment or reaction counts, but **we are not saying no** to your request. (We know some of these have been open since before I had gray hair. And I've had gray hair for a LONG time.)

**Could you do us a favor? If this icon request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
→ https://github.com/FortAwesome/Font-Awesome/discussions/new?category=icon-requests

And listen. We know this is annoying. We would batch convert them to discussions if we could, but Github's removed that ability earlier this year. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worries at all. I love doing things twice.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. This request already HAS an auto-response saying that you're refining the workflow. What do you mean you're refining your refinement? That's a bunch of bull-phooey.
* No biggie. Thanks for letting us know what's up.

**Comment by sensibleworld** [comment]
Released in 7.2: https://fontawesome.com/icons/wardrobe?f=classic&s=solid
